package.json

```diff
@@ -35,23 +34,20 @@
     "watch:sass": "npm run css-compile -- --watch"
   },
   "peerDependencies": {
-    "bootstrap": "5.2.2",
-    "popper.js": "1.16.1"
+    "bootstrap": "~5.2.0"
```

Comment:
This is a downstream dependency for consuming apps - and they should not be pinned to 5.2.2 because they can use other 5.2 versions, and [already are](https://github.com/techfromsage/user-utils-app/blob/8f99971db360b75228a63b6d63a9ffbf8f61b601/package.json#L41).